code/modules/SCP/SCPs/SCP-106.dm

```diff
@@ -58,7 +58,7 @@
 	)
 
 	SCP.min_time = 40 MINUTES
-	SCP.min_playercount = 20
+	SCP.min_playercount = 35 //have at least 1 guard that set up the generators so they dont escape and murder everyobe
```

Comment:
```suggestion
	SCP.min_playercount = 25
```
this is ridiculous put it to 20-25. the fact you made it more than 2427 is really dumb

Reply:
106 is a set-and-forget SCP with the wiring changes I made; it will only breach in case HCZ and engineering is utterly incompetent